src/module-3/online-store/scss/components/_category.scss

```diff
@@ -0,0 +1,53 @@
+.category {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+
+  &__item {
+    width: 100%;
+
+    &_info {
+      display: flex;
+      justify-content: space-between;
+      flex-wrap: nowrap;
+    }
+  }
+
+  &-col {
+  display: flex;
+  flex-direction: column;
+}
+
+  &__divider {
+    height: 1px;
+    margin-top: 28px;
+    margin-bottom: 33px;
+    width: 100%;
+  }
+
+  &__headline {
+    line-height: 22px;
+    margin-bottom: 25px;
+  }
+}
+
+
+.category__radio-btn:checked + label:after,
```

Comment:
that could be written in more compact manner using nested for all `.category__radio-btn` labels (you use here both checked and not checked modifiers, is there some 3rd variant?)